pyuvdata/uvdata/corr_fits_src/corr_fits.pyx

```diff
@@ -37,15 +37,18 @@ cpdef dict input_output_mapping():
 
   return pfb_inputs_to_outputs
 
-cpdef tuple generate_map(
+@cython.boundscheck(False)
+@cython.wraparound(False)
+cpdef void generate_map(
```

Comment:
It might be worth mentioning (either in a comment or a docstring) that `map_inds` and `conj` are update "in-place", so to speak, without explicitly being returned to the user. I know the `void` return type and name hints at that, but it might be good to be explicit (for people like me who got confused at first...).